vnext/JSI/Shared/ChakraRuntime.h

```diff
@@ -248,15 +239,35 @@ class ChakraRuntime : public facebook::jsi::Runtime {
   ChakraObjectRef ToChakraObjectRef(const facebook::jsi::Value &value);
   std::vector<ChakraObjectRef> ToChakraObjectRefs(const facebook::jsi::Value *value, size_t count);
 
-  // Host function and host object helpers
+  // Convenience functions for property access.
+  ChakraObjectRef GetProperty(const ChakraObjectRef &obj, const ChakraObjectRef &id);
+  inline ChakraObjectRef GetProperty(const ChakraObjectRef &obj, const char *const name) {
```

Comment:
>  i [](start = 0, length = 3)

(nit) can you add empty line here for readability?

Reply:
Same with previous comment.

---
In reply to: [341260719](https://github.com/microsoft/react-native-windows/pull/3568#discussion_r341260719) [](ancestors = 341260719)